dist/leaflet.css

```diff
@@ -223,6 +223,10 @@ svg.leaflet-image-layer.leaflet-interactive path {
 .leaflet-container a {
 	color: #0078A8;
 	}
+/* prevent showing outline-box on Chromium when clicking on a vector with a tooltip */
+.leaflet-container path.leaflet-interactive:focus:not(:focus-visible) {

```

Comment:
I think we can omit `.leaflet-container `, the remaining selector is specific enough.